installation/managed-identity-authentication.md

```diff
@@ -0,0 +1,59 @@
+---
+description: This article explains how to upgrade your authentication to Managed Identity. 
+---
+
+# Managed Identity Authentication
+
+Instead of using SQL authentication, you are now able to **use Managed Identity authentication with SysKit Point**.
+
+In order to enable Managed Identity authentication to access data in the SysKit Point, complete the following:
+
+* **Navigate to [Azure Portal](https://portal.azure.com/) and open the SysKit Point resource group**
+* **Click the Managed Identity resource**; located under Resources and marked as Managed Identity in the Type column
+* **Copy the Client ID**; located on the right side of the Managed Identity resource screen 
+   * This will be needed later, so store the copied text somewhere it will be easy to get it later
+* Return back to the **SysKit Point resource group** page
+* **Click the Key Vault resource**; located under Resources and marked as Key Vault in the Type column
+* **Select the Access Policies option**; located on the left side of the screen
+* Click the **Create** button at the top of the screen
+  * This will provide the option to **Create an access policy**
+* Under **Configure from a template** select the **Secret Management** option and click on **Next**
+* For **Find Principal select the Admin** that is completing the Managed Identity authentication process and click **Next**
+* The Application section can be skipped, so **select Next**
+* Under **Review + Create, select the principal name** of the person that was selected in the previous step
+* **Click Create** when finished
+
+When this is set up, **the Access Policies screen should now show the name of the person selected** during the creation process under the **User** section. If this was completed successfully, proceed to the following steps:
+
+* On the Key Vault resource, **select Netwoking**; located on the left side under Settings
+* Under Firewall, **click Add your client IP address** and **paste your public IP address** here
+* **Click Apply** when finished
+* **Select Secrets**; located on the left side under Objects
+* **Select the PointDatabase option** from the list of available names
+* On the PointDatabase screen, **click the New Version button**
+* On the form that opens, you will need to enter the following text as **Secret value**
+
+ `Data Source=tcp:{servername}.database.windows.net,1433;Initial Catalog=SysKitPointDB;User Id="{managed identity client id}";Authentication=Active Directory Managed Identity;`
+
+{% hint style="warning" %}
+**IMPORTANT!**  
+This part of the text: User Id="{**managed identity client id**} needs to be changed. The Client ID you previously copied from your Managed Identity resource should be placed where the text says **managed identity client id**. 
```

Comment:
ne mijenja se samo taj dio texta, nego i {servername}, znači treba i na početku otvoriti SQL server resurs i tamo uzeti Serrver name. S tim da bi tu onda stavio da umjesto {servername}.database.windows.net piše samo {serverName}, jer se u na SQL resursu lako može kopirati na taj način https://sharexuploadsstorage.blob.core.windows.net/sharexuploads/ShareX/2022/10/IMESARIC_06.10.2022_15.38.36.png